dolphinscheduler-standalone-server/src/main/resources/application.yaml

```diff
@@ -164,22 +164,17 @@ worker:
   #Scenes to be used for distributed users.For example,users created by FreeIpa are stored in LDAP.This parameter only applies to Linux, When this parameter is true, worker.tenant.auto.create has no effect and will not automatically create tenants.
   tenant-distributed-user: false
   # worker max cpuload avg, only higher than the system cpu load average, worker server can be dispatched tasks. default value -1: the number of cpu cores * 2
-  max-cpu-load-avg: -1
+  max-cpu-load-avg: 80
   # worker reserved memory, only lower than system available memory, worker server can be dispatched tasks. default value 0.3, the unit is G
-  reserved-memory: 0.3
-  # default worker groups separated by comma, like 'worker.groups=default,test'
-  groups:
-    - default
-  # alert server listen host
-  alert-listen-host: localhost
-  alert-listen-port: 50052
```

Comment:
Can you review these changed configurations in the Helm Chart and see whether we have these there, if so remove them as well?

Reply:
I find these exist in `kubernetes` and `terraform`, and remove these, please help to check again.